### PR TITLE
Fires debug log while authInMemory is true

### PR DESCRIPTION
**Pull request by herpiko**

### CODE REVIEW

**Comment by emhaye** [comment]
:+1:

**Comment by herpiko** [comment]
The most curious PR I've ever done. :smile:

File: index.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ var User = function(server, options, next) {
   var getCredentials = function(id, callback) {
     var checkToken = function(id, cb){
       if (options.authInMemory) {
+        console.log(inMemoryTokens);
         var result = false;
         if (inMemoryTokens[id]) {
```
Comment (mdamt) [inline index.js:76]: 1. Remove all console.log
2. Replace A to B

A:

```
         if (inMemoryTokens[id]) {
            result = inMemoryTokens[id];
         }
```

B:

```
            result = inMemoryTokens[id];

```

**Comment by mdamt** [comment]
-1

**Pull request closed without merging**